factory/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 3.2.0
 
+* Updated default node version from `20.8.1` to `20.9.0`. Addressed in [#987](https://github.com/cypress-io/cypress-docker-images/pull/987)
```

Comment:
For this change to release you need to update the FACTORY_VERSION. This change wouldn't be present in 3.2.0 so the change log needs to be updated too.